frontend/src/components/login-page/auth/fields/username-field.tsx

```diff
@@ -8,19 +8,26 @@ import React from 'react'
 import { Form } from 'react-bootstrap'
 import { useTranslation } from 'react-i18next'
 
+export interface UsernameFieldProps extends AuthFieldProps {
+  value: string
+}
+
+//TODO: This should be replaced with the common username component
```

Comment:
Is there an issue for this?

Reply:
https://github.com/hedgedoc/hedgedoc/issues/4128